Build error on MacOS

Hey all!

I've started getting a build error on MacOS since I pulled master. Seems to be related to `base64_decode_sextet`.

```/Users/stephen/work/ruby/forks/natalie/include -I /Users/stephen/work/ruby/forks/natalie/build/onigmo/include -I /Users/stephen/work/ruby/forks/natalie/build/natalie_parser/include -std=c++17 -c -o build/string_object.cpp.o src/string_object.cpp
In file included from src/string_object.cpp:3:
/Users/stephen/work/ruby/forks/natalie/include/natalie/string_unpacker.hpp:253:26: error: implicit instantiation of undefined template 'std::array<unsigned char, 3>'
            auto chars = base64_decode_sextet(a, b, c, d);
                         ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/__tuple:219:64: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TEMPLATE_VIS array;
                                                               ^
In file included from src/string_object.cpp:3:
/Users/stephen/work/ruby/forks/natalie/include/natalie/string_unpacker.hpp:342:34: error: implicit instantiation of undefined template 'std::array<unsigned char, 3>'
    std::array<unsigned char, 3> base64_decode_sextet(unsigned char a, unsigned char b, unsigned char c, unsigned char d) {
                                 ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/__tuple:219:64: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TEMPLATE_VIS array;
                                                               ^
In file included from src/string_object.cpp:3:
/Users/stephen/work/ruby/forks/natalie/include/natalie/string_unpacker.hpp:354:16: error: implicit instantiation of undefined template 'std::array<unsigned char, 3>'
        return { out_a, out_b, out_c };
               ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/c++/v1/__tuple:219:64: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TEMPLATE_VIS array;
                                                               ^
3 errors generated.
rake aborted!
```

--- a/include/natalie/string_unpacker.hpp
+++ b/include/natalie/string_unpacker.hpp
@@ -2,6 +2,7 @@
 
 #include "natalie.hpp"
 #include "natalie/array_packer/tokenizer.hpp"
+#include <array>
 
 namespace Natalie {
 

